main.ledger

```diff
@@ -6873,7 +6927,265 @@
+    Assets:Chase:Checking
+    ; Receipt: 65f9266f8d608f53f93180daaed76cd5.pdf
+
+2018/01/30 Deposit
+    Assets:Chase:Checking                           $25,000.00
+    Income:Fundraising
+    ; Receipt: 65f9266f8d608f53f93180daaed76cd5.pdf
+
+2018/01/31 Max Wofford
+    Expenses:Operating:Staff:Salary                  $3,907.00
+    Assets:Chase:Checking
+    ; Receipt: 65f9266f8d608f53f93180daaed76cd5.pdf
+
+2018/01/31 Gusto
+    ;Tax and Net Pay in Gusto
+    Expenses:Operating:                                $168.66
+    Expenses:Operating:                                 $36.54
+    Assets:Chase:Checking
+    ; Receipt: 65f9266f8d608f53f93180daaed76cd5.pdf
+
+2018/01/31 Starbucks
+    Expenses:Fundraising:Food                            $2.45
+    Liabilities:Reimbursement:Zach Latta
+    ; Receipt: a0600c7a55844d28cc427ebd1d0d962d.pdf
+
+2018/01/31 Cafe Borrone
+    Expenses:Fundraising:Food                           $15.23
+    Liabilities:Reimbursement:Zach Latta
+    ; Receipt: a0600c7a55844d28cc427ebd1d0d962d.pdf
+
+2018/02/03 Gusto
+    Expenses:Operating:Software                         $51.00
+    Assets:Chase:Checking
+    ; Receipt: e51785afc2b35c8856cda82694169b1f.jpg
+
+2018/02/02 Stripe
+    Assets:Chase:Checking                             $6234.27
+    Income:Website Donations
+    ; Receipt: e51785afc2b35c8856cda82694169b1f.jpg
+
+2018/02/01 Caltrain
+    Expenses:Fundraising:Transportation:Ground          $15.40
+    Liabilities:Reimbursement:Zach Latta
+    ; Receipt: bdc539c8f878a24da5ecd40b87b8f0a9.pdf
+
+2018/02/01 Heroku
+    Expenses:Operating:Hosting                         $241.35
+    Liabilities:Reimbursement:Zach Latta
+    ; Receipt: 44c72c18ef91db8d7b7d367dd14997bc.pdf
+
+2018/02/01 Papertrail
+    Expenses:Operating:Software                          $7.00
+    Liabilities:Reimbursement:Zach Latta
+    ; Receipt: 9fbdf99228f33ea410d8a693879e8d29.pdf
+
+2018/02/02 Gusto
+    Expenses:Operating:Software                         $57.00
+    Assets:Chase:Checking
+    ; Receipt: e51785afc2b35c8856cda82694169b1f.jpg
+
+
+2018/02/05 Namecheap
+    Expenses:Operating:Hosting                          $70.88
+    Liabilities:Reimbursement:Zach Latta
+    ; Receipt: 128bc8a951953ecdd9e11267db74e911.pdf
+
+2018/02/11 TransferWise
+    Expenses:Operating:Software                        $442.93
+    Assets:Chase:Checking
+    ; Receipt: e51785afc2b35c8856cda82694169b1f.jpg
+
+2018/02/06 Walgreens
+    Expenses:Operating:Office:Supplies                   $7.19
+    Liabilities:Reimbursement:Max Wofford
+    ; Receipt: 263d94220318ecb185dc92686fd44428.jpeg
+
+2018/02/08 Streak
+    Expenses:Operating:Software                        $156.00
+    Liabilities:Reimbursement:Zach Latta
+    ; Receipt: 00a6755990ca56f88e3f4a91f9767ffd.pdf
+
+2018/02/09 Virgin America
+    Expenses:Operating:Transportation:Air               $45.30
+    Liabilities:Reimbursement:Zach Latta
+    ; Receipt: 8cf123ac540e0be1cda46a067f0f1f0b.pdf
```

Comment:
I am so sorry - this is a mistake on my part. This is a personal expense, not Hack Club.